src/lib/datepicker/datepicker-input.ts

```diff
@@ -152,7 +174,7 @@ export class MdDatepickerInput<D> implements AfterContentInit, ControlValueAcces
       Validators.compose([this._minValidator, this._maxValidator, this._filterValidator]);
 
   constructor(
-      private _elementRef: ElementRef,
+      public _elementRef: ElementRef,
```

Comment:
Have you considered passing the element and the value to `MdDatepickerInputEvent` separately (instead of `this`), so you can keep `_elementRef` private?